src/Pages/Overview/App.js

```diff
@@ -0,0 +1,343 @@
+    user !== false &&
+      this.setState(
+        { toggleDelete: !this.state.toggleDelete },
+        this.setState({ selectedUser: { ...user } })
+      )
+  }
+
+  callDatabase () {
+    axios
+      // get all user!
+      .post('/api/user/search', {
+        // don't need email
+        token: this.state.authToken
+      })
+      .then(result => {
+        if (result.status >= 200 && result.status < 300) {
+          this.setState({ users: result.data })
+        }
+      })
+      .catch(err => {
+        console.log(err)
+      })
+  }
+
+  updateSearch (event) {
+    this.setState({ search: event.target.value })
+  }
+
+  /*
+  Delete api
+  parameter: Json object of object to be deleted
+  */
+  deleteUser (user) {
+    // if they decide to delete themselves: delete->logout
+    if (user.email === this.state.currentUser) {
+      axios
+        .post('/api/user/delete', {
+          token: this.state.authToken,
+          email: user.email
+        })
+        .then(result => {
+          this.callDatabase() // reload database
+        })
+        .catch(err => {
+          console.log(err)
+        })
+
+      // logout
+      window.localStorage.removeItem('jwtToken')
+      window.location.reload()
+      return window.alert('Self-deprecation is an art')
+    } else {
+      axios
+        .post('/api/user/delete', {
+          token: this.state.authToken,
+          email: user.email
+        })
+        .then(result => {
+          this.callDatabase() // reload database
+        })
+        .catch(err => {
+          console.log(err)
+        })
+    }
+    this.handleToggleDelete()
+  }
+
+  // simply filter array by name
+  search () {
+    const search = this.state.search.trim().toLowerCase()
+    return search !== null || search !== ''
+      ? this.state.users.filter(data =>
+        data.firstName.toLowerCase().includes(search.toLowerCase())
+      )
+      : this.state.users
+  }
+
+  // Return check mark if true
+  marks (sign) {
+    return sign === true ? (
+      <svg width='24' height='24' viewBox='0 0 24 24' style={{ fill: 'GREEN' }}>
+        <path d='M20.285 2l-11.285 11.567-5.286-5.011-3.714 3.716 9 8.728 15-15.285z' />
+      </svg>
+    ) : (
+      <svg width='24' height='24' viewBox='0 0 24 24' style={{ fill: 'RED' }}>
+        <path d='M24 20.188l-8.315-8.209 8.2-8.282-3.697-3.697-8.212 8.318-8.31-8.203-3.666 3.666 8.321 8.24-8.206 8.313 3.666 3.666 8.237-8.318 8.285 8.203z' />
+      </svg>
+    )
+  }
+
+  /*
+  translating access Level into role
+  The access level is defined as follows:
+  -2: Ban
+  -1: Pending
+   0: Member
+   1: Officer
+   2: Admin
+  */
+  roleTranslator (level) {
+    switch (level) {
+      case -2:
+        return 'Ban'
+      case 0:
+        return 'Member'
+      case 1:
+        return 'Officer'
+      case 2:
+        return 'Admin'
+      default:
+        return 'Pending'
+    }
+  }
+
+  // displaying children of board
+  display (user, index) {
```

Comment:
this should be it's own component. see if you can make a component that you can use across overview and admin.